src/components/App/filters/filters.js

```diff
@@ -68,3 +76,230 @@ export function formatVersion (string) {
 export function globalize (translationId, interpolationOps) {
   return i18n.t(translationId, interpolationOps)
 }
+
+export function formatDate (value) {
+  return globalize('formats.date', { value })
+}
+
+export function yesNoFilter (value) {
+  if (value) {
+    return globalize('filters.yes-no-filter.yes')
+  } else {
+    return globalize('filters.yes-no-filter.no')
+  }
+}
+
+export function assetRequestStatesFilter (value) {
+  let translationId = ''
+  switch (value) {
+    case CREATE_ASSET_REQUEST_STATES.pending.codeVerbose: {
+      translationId = 'filters.asset-request-states-filter.pending'
+      break
+    }
+    case CREATE_ASSET_REQUEST_STATES.canceled.codeVerbose: {
+      translationId = 'filters.asset-request-states-filter.canceled'
+      break
+    }
+    case CREATE_ASSET_REQUEST_STATES.approved.codeVerbose: {
+      translationId = 'filters.asset-request-states-filter.approved'
+      break
+    }
+    case CREATE_ASSET_REQUEST_STATES.rejected.codeVerbose: {
+      translationId = 'filters.asset-request-states-filter.rejected'
+      break
+    }
+    case CREATE_ASSET_REQUEST_STATES.permanentlyRejected.codeVerbose: {
+      translationId = 'filters.asset-request-states-filter.perm-rejected'
+      break
+    }
+    default: {
+      translationId = 'filters.asset-request-states-filter.default'
+      break
+    }
+  }
+  return globalize(translationId)
+}
+
+export function assetRequestTypesFilter (value) {
+  let translationId = ''
+  switch (value) {
+    case ASSET_REQUEST_TYPES.createAsset: {
+      translationId = 'filters.asset-request-types-filter.create'
+      break
+    }
+    case ASSET_REQUEST_TYPES.updateAsset: {
+      translationId = 'filters.asset-request-types-filter.update'
+      break
+    }
+    default: {
+      translationId = 'filters.asset-request-types-filter.default'
+      break
+    }
+  }
+  return globalize(translationId)
+}
+
+export function assetPoliciesVerboseFilter (value) {
+  let translationId = ''
+  switch (value) {
+    case ASSET_POLICIES_VERBOSE[ base.xdr.AssetPolicy.transferable().value ]: {
+      translationId = 'filters.asset-policies-verbose-filter.transferable'
+      break
+    }
+    case ASSET_POLICIES_VERBOSE[ base.xdr.AssetPolicy.baseAsset().value ]: {
```

Comment:
it is superfluous `ASSET_POLICIES_VERBOSE `

Reply:
fixed